extension/js/application/states/home/home.js

```diff
@@ -171,8 +171,13 @@ app.controller('HomeCtrl', function ($scope, $state, popupGitFactory, $timeout,
 
 	//log out
 	$scope.logout = function () {
-		popupGitFactory.logout().then(function(){
+		popupGitFactory.logout().then(function(res){
```

Comment:
capitalize Factory names.  they represent 'classes' in javascript, kind of like constructor functions